setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(name='pypif-sdk',
-      version='2.1.0',
+      version='2.1.1',
```

Comment:
This includes a new method, so I think it makes more sense as a minor version bump (i.e. 2.2.0).